zbus_macros/src/iface.rs

```diff
@@ -1017,7 +1017,7 @@ fn get_args_from_inputs(
                 let header_arg = &input.pat;
 
                 header_arg_decl = match method_type {
-                    MethodType::Property(_) => Some(quote! { let #header_arg = header.clone(); }),
+                    MethodType::Property(_) => Some(quote! { let #header_arg = header.cloned(); }),
```

Comment:
As discussed in the PM, we want to use the full paths in the generated code: `::std::Option::cloned(header);`

Reply:
AFAICT (unless GH is lying to me), this is not resolved. :thinking:

Reply:
No, sorry, I marked it resolved before actually pushing. I do that to keep track of my work, but since obviously this is not the workflow you're using I will refrain from doing that in future PRs.

Reply:
>  do that to keep track of my work, but since obviously this is not the workflow you're using I will refrain from doing that in future PRs.

I think it's a good thing you track your progress like this. :+1: I do suggest doing it only after pushing, not only for the benefit of the reviewer but also to ensure that you didn't forget to push. :)

Reply:
Hmm.. it's still not resolved btw but the conversation was resolved? I'm getting very confused now so I'll take a break..

Reply:
GH was showing me old commits for some reason. :thinking:

Reply:
> GH was showing me old commits for some reason. 🤔

No, you ended up putting it in the wrong (second) commit. :) This is a fixup of the first commit.

Reply:
A break sounds good :slightly_smiling_face:.

Here's the commit: https://github.com/dbus2/zbus/pull/1179/commits/e3e0d1105a12415adf65ef12ac0d685502618dbf#diff-43a2e53b8de7776ea27dd141513cf075f24db06d6612dd519ff04780a301807cR1039

I don't know what GitHub's thinking.

Reply:
> No, you ended up putting it in the wrong (second) commit. :) This is a fixup of the first commit.

You're right, sorry! Fixed.